apps/web/components/AppListCard.tsx

```diff
@@ -96,7 +97,7 @@ export default function AppListCard(props: AppListCardProps) {
                   className="mr-2"
                   alt={credentialOwner.name || "Nameless"}
                   size="xs"
-                  imageSrc={credentialOwner.avatar}
+                  imageSrc={getPlaceholderAvatar(credentialOwner.avatar, credentialOwner?.name as string)}
```

Comment:
Fix placeholder avatar for teams

Reply:
<img width="647" alt="Screenshot 2023-07-27 at 2 45 52 PM" src="https://github.com/calcom/cal.com/assets/1780212/443bb8a0-dd1f-482b-addc-a5a7671dbe95">
Possible Followup: Avatar component has fallback image support but it uses gravatar.com maybe we can just switch it with ui-avatars.com and automatically enable fallback.
<img width="1206" alt="Screenshot 2023-07-27 at 2 45 16 PM" src="https://github.com/calcom/cal.com/assets/1780212/30ae67c0-751d-410c-97a9-7439caaf5a85">

Reply:
Created a ticket to track this.https://github.com/calcom/cal.com/issues/10475